src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int NUMBER_LIMIT = 101;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(NUMBER_LIMIT));
```

Comment:
Create "Random" and "ColorSupplier" as class fields. Don't create new Random and new ColorSupplier every time when method getRandomBall() is invoked